pfsspy/utils.py

```diff
@@ -293,3 +293,68 @@ def car_to_cea(m, method='interp'):
                          return_footprint=False)
 
     return sunpy.map.Map(data_out, header_out)
+
+def roll_map(m, lh_edge_lon = 0.0, method='interp'):
```

Comment:
`lh_edge_lon` should be an astropy quantity - see https://docs.astropy.org/en/stable/api/astropy.units.quantity_input.html for more info on how to do this.

Reply:
decorator added